File: sdkwork-backend-sdk-csharp/Models/PlusOrderWorkerDispatchProfileVO.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Backend.Models
{
    public class PlusOrderWorkerDispatchProfileVO
    {
        public string? CreatedAt { get; set; }
        public string? UpdatedAt { get; set; }
        public int? Id { get; set; }
        public int? UserId { get; set; }
        public string? RatingLevel { get; set; }
        public bool? Enabled { get; set; }
        public int? GlobalMaxInProgress { get; set; }
        public OrderWorkerDispatchMetadata? Metadata { get; set; }
    }
}

File: sdkwork-backend-sdk-kotlin/src/main/kotlin/com/sdkwork/backend/PlusOrderWorkerDispatchProfileVO.kt
package com.sdkwork.backend

data class PlusOrderWorkerDispatchProfileVO(
    val createdAt: String? = null,
    val updatedAt: String? = null,
    val id: Int? = null,
    val userId: Int? = null,
    val ratingLevel: String? = null,
    val enabled: Boolean? = null,
    val globalMaxInProgress: Int? = null,
    val metadata: OrderWorkerDispatchMetadata? = null
)
